src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -9,16 +8,27 @@
 import net.minecraftforge.common.AchievementPage;
 
 /**
- * Achievement collection
+ * Achievement collection.
```

Comment:
Imporve this javadoc b writing multiple sentences explaining ehat this class is doing.